### ISSUE TITLE
Cache Get Data - L1 Cache Error

### ISSUE THREAD

**Issue by XinYenFon** [SimpleMachines/SMF#270]
In latest nightly build when i try to enable cache level 1 i get this error.

8: Undefined variable: value - Load.php (Line: 2802)

```
if (function_exists('call_integration_hook'))
    call_integration_hook('cache_get_data', array($key, $ttl, $value));
```

**Comment by colinschoen** [comment]
Did this not happen on the alpha release or previous builds? (I will test when I get home)

**Comment by emanuele45** [comment]
Don't know, noticed me too yesterday evening when I decided to give the cache a try...
Fix could simply be:
`if (function_exists('call_integration_hook') && isset($value))`

**Comment by emanuele45** [comment]
I fixed this in 7ea792efecb25445f31763e5cd409f3cf4cc0382
